### PR TITLE
add README.rst on partner_address_street3

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

File: partner_address_street3/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+partner address street3
```
Comment (pedrobaeza) [inline partner_address_street3/README.rst]: Put as title something like "Third address field in partners"

**Comment by gurneyalex** [comment]
see also #126

**Comment by lepistone** [comment]
:+1: thanks

**Comment by pedrobaeza** [comment]
:+1:

**Comment by PCatinean** [comment]
:+1:

**Pull request merged**